tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Go To Config-Maps
+    Select Redhat-ODS-Applications
+    Select Rhods-Groups-Config
+    Wait Until Page Contains Element    xpath=//*[@class='co-pre-wrap co-copy-to-clipboard__text']
```

Comment:
I would put line 17,18,19 in a separated keyword with a more user-friendly name, for example, `Verify Groups ConfigMap contains The Expected OCP Groups`